hokusai/cli/review_app.py

```diff
@@ -0,0 +1,35 @@
+import click
+from shutil import copyfile
+import yaml
+
+import hokusai
+
+from hokusai.cli.staging import staging, KUBE_CONTEXT
+from hokusai.lib.common import set_verbosity, CONTEXT_SETTINGS
+from hokusai.lib.config import config
+
+@staging.group()
+def review_app(context_settings=CONTEXT_SETTINGS):
+  """Manage/Create review apps"""
+  pass
+
+@review_app.command(context_settings=CONTEXT_SETTINGS)
+@click.argument('app_name', type=click.STRING)
+@click.option('-v', '--verbose', type=click.BOOL, is_flag=True, help='Verbose output')
+def create(app_name, verbose):
+  set_verbosity(verbose)
+  # copy staging yml file
+  create_new_app_yaml('hokusai/staging.yml', app_name)
```

Comment:
[Ceiling cat] Seems like if the review app is by definition based on the staging environment, then maybe the proposed cli syntax could drop that detail? So just <tt>hokusai ~staging~ review_app create</tt>?

Reply:
good point, that actually makes it easier for later if we wanted to do things like `hokusai review_app deploy <app name>`.